config/liota.conf

```diff
@@ -16,7 +16,7 @@ enable_reboot_getprop = False
 # The number of attempts to retry in order to read iotcc.json files
 iotcc_load_retry = 3
 # Configurable queue timeout in order to wait for the response messages from IOTCC
-iotcc_response_timeout = 300
+iotcc_response_timeout = 600
```

Comment:
I believe this is part of a separate pull request. Please remove it from here.

Reply:
Raised a new PR 
https://github.com/vmware/liota/pull/206
addressing all the review comments.

I have closed this PR due to merge conflict.